### PR TITLE
FDR mount drives and controller models sampled at 8kHz

### PR DESCRIPTION
**Pull request by rodrigo-romano**

### CODE REVIEW

File: mount/controller/build.rs [hunk 1]
```diff
@@ -9,13 +9,13 @@ fn main() {
         Ok(val) => match val.as_str() {
             "MOUNT_PDR_8kHz" => "sys_pdr_za30_8k",
             "MOUNT_FDR_1kHz" => "sys_fdr_1k",
-            //"MOUNT_FDR_8kHz" => "sys_fdr_8k",
+            "MOUNT_FDR_8kHz" => "sys_fdr_8k",
             "MOUNT_FDR_1kHz-az17Hz" => "sys_fdr_1k-az17Hz",
             value => panic!("expected one of the following `MOUNT_MODEL`: `MOUNT_PDR_8kHz`, `MOUNT_FDR_1kHz` or `MOUNT_FDR_1kHz-az17Hz`, found {value}"),
```
Comment (rconan) [inline mount/controller/build.rs]: can you add `MOUNT_FDR_8kHz` to the list of `MOUNT_MODEL`s

File: mount/driver/build.rs [hunk 2]
```diff
@@ -9,12 +9,12 @@ fn main() {
         Ok(val) => match val.as_str() {
             "MOUNT_PDR_8kHz" => "sys_pdr_8k-linear",
             "MOUNT_FDR_1kHz" | "MOUNT_FDR_1kHz-az17Hz"=> "sys_fdr_1k",
-            //"MOUNT_FDR_8kHz" => "sys_fdr_8k",
+            "MOUNT_FDR_8kHz" => "sys_fdr_8k",
             value => panic!("expected one of the following `MOUNT_MODEL`: `MOUNT_PDR_8kHz`, `MOUNT_FDR_1kHz` or `MOUNT_FDR_1kHz-az17Hz`, found {value}"),
```
Comment (rconan) [inline mount/driver/build.rs]: can you add MOUNT_FDR_8kHz to the list of MOUNT_MODELs

Comment (rodrigo-romano) [reply mount/driver/build.rs]: Commit [7d79522](https://github.com/rconan/gmt-controllers/pull/8/commits/7d79522357d274d6423cf55e1a1e4862b066a13a) addresses  that.

**Review by rconan — APPROVED** [review]

**Comment by rodrigo-romano** [comment]
The last [commit](https://github.com/rconan/gmt-controllers/pull/8/commits/5e1df0242392fc783473e8279e84231ac02c5bde) bumps the mount controller and driver version to 1.1.0 and updates the corresponding readme files with the FDR 8kHz mount control model option.

File: Cargo.toml [hunk 3]
```diff
@@ -16,9 +16,9 @@ resolver = "2"
 authors = ["Rod Conan <rconan@gmto.org>"]
 edition = "2021"
 license = "MIT"
-description = "Giant Magellan Telescope primary mirror control system"
-repository = "https://github.com/rconan/m1-ctrl"
-documentation = "https://docs.rs/gmt_m1-ctrl"
+description = "Giant Magellan Telescope C-code control system models"
```
Comment (rconan) [inline Cargo.toml:19]: Giant Magellan Telescope Simulink to C-code control system models

**Review by rconan — APPROVED** [review]

**Review by rconan — APPROVED** [review]

**Pull request merged**